build/Settings.cs

```diff
@@ -48,8 +48,8 @@ private static string config(string @default = null, [CallerMemberName] string k
 
         public static readonly Dictionary<string, string[]> ToolsRuntimeToPowershellRuntimes = new Dictionary<string, string[]>
         {
-            { "win-x86", new [] { "win-x86", "win", "win10-x86", "win8-x86", "win81-x86", "win7-x86" } },
-            { "win-x64", new [] { "win-x64", "win", "win10-x64", "win8-x64", "win81-x64", "win7-x64" } },
+            { "win-x86", new [] { "win-x86", "win", "win10-x86", "win8-x86", "win81-x86", "win7-x86", "win-x64", "win", "win10-x64", "win8-x64", "win81-x64", "win7-x64" } },
```

Comment:
There is duplicate entry for "win". Also, since both x86 and x64 will have the same values. We can create one list and reference it "win-x64" and "win-x86".